README.md

```diff
@@ -85,6 +107,20 @@ If what you want are unbound exponential backoffs, you can initiate them with:
 
 And still use them as usual. The increments will have no upper limit.
 
+## State Backoffs with jitter
+
+You can enable a jitter based incremental backoff by calling `jitter/1`
+that toggles the state of the backoff:
+
+    1> B0 = backoff:init(1, 10000, self(), put_it).
```

Comment:
I would use `init/2` for this one; the stateful ones with messages are only introduced in the "Timeout Events" sections and this new form is a bit confusing to read in the doc :)